components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java

```diff
@@ -74,7 +76,10 @@ public Label addLabel(String tenantDomain, Label label) throws APIManagementExce
      * @param labelId Label identifier
      * @throws APIManagementException If failed to delete label
      */
-    public void deleteLabel(String labelId) throws APIManagementException{
+    public void deleteLabel(String user, String labelId) throws APIManagementException {
+        if (isAttachedLabel(user, labelId)) {
+            APIUtil.handleException("Unable to delete label. Label is already attached to a API");
```

Comment:
```suggestion
            APIUtil.handleException("Unable to delete the label. It is attached to an API");
```

Reply:
fixed

Reply:
@nalakas Let's have the practice to add the commit hash too when mentioning an comment/issue/suggestion as fixed. i.e. Fixed with 46c6ec9